### ISSUE TITLE
there is some error on docs page

### ISSUE THREAD

**Issue by fmnisme** [saltstack/salt#29998]
https://docs.saltstack.com/en/latest/ref/netapi/all/salt.netapi.rest_cherrypy.html#keys

```Mock out specified imports
This allows autodoc to do its thing without having oodles of req'd installed libs. This doesn't work with import * imports.

http://read-the-docs.readthedocs.org/en/latest/faq.html#i-get-import-errors-on-libraries-that-depend-on-c-modules
```

**Comment by cachedout** [comment]
@jacobhammons or @whiteinge Can one of you took a look here? This looks like something is very broken with the way Sphinx is parsing that docstring.

**Comment by whiteinge** [comment]
@fmnisme thanks for the report. This issue will be resolved when #28540 is resolved. I'll follow up with Jacob.

**Comment by cachedout** [comment]
I'm going to close this as a duplicate then. Please follow #28540 for status.
